### DISCUSSION TITLE
"Show in Tree" only works if sorting is alphabetical

### DISCUSSION THREAD

**Discussion by polymed** [pimcore/pimcore#8474] [Q&A / Support] [answered]
Hi guys, 

when I am on an object and I run the "Show in Tree" function, it only works if the objects are sorted alphabetically. Is that the way it is supposed to be?

Best regards

Pimcore-Version: 6.8.10

**Comment by dvesh3** [comment, marked as answer]
bug reported. please see https://github.com/pimcore/pimcore/issues/8476

**Reply by polymed** [reply]
Thank you
